rustls/src/msgs/message.rs

```diff
@@ -211,19 +228,6 @@ impl<'a> BorrowedOpaqueMessage<'a> {
             payload,
         })
     }
-
-    pub(crate) fn into_owned(self) -> OpaqueMessage {
-        let Self {
-            typ,
-            version,
-            payload,
-        } = self;
-        OpaqueMessage {
-            typ,
-            version,
-            payload: Payload::new(&*payload),
```

Comment:
It looks like while this was moved the `&*` was dropped, can you fix that in the commit that added `into_owned` on `BorrowedOpaqueMessage` instead?